content/getting-started/multiDeviceMultiSig/_index.en.md

```diff
@@ -0,0 +1,175 @@
+
+{{< /tab >}}
+
+{{< tab "Mobile" >}}
+
+{{< notice note >}}
+If you did not have all your keys available for the initial setup, you can continue from another device to continue the setup process. It is assumed that you have started to create a wallet. If you still have a pending configuration notification in your chat screen you can continue on your laptop here. If you do not, you should refer to the desktop tab to start a configuration.
+{{< /notice >}}
+
+1. Open your Nunchuk mobile app and navigate to the chat screen. You should see your wallet config notification here waiting for the last key!<br/><br/>
+
+<img src = mobilePendingKey.png
+    alt = "mobile app chat screen (Screenshot)">
+
+2. Click on the "View config" button.<br/><br/>
+
+3. Click the "Continue to assign keys" button at the bottom of the screen.<br/><br/>
+
+<img src = mobileContinueAssignKeys.png
+    alt = "Wallet configuration (Screenshot)">
+
+4. You can now assign the last key (in my case, it is the Mobile Key).<br/><br/>
+
+5. Click the "Assign keys and send invite" to finalize your wallet with the last key.<br/><br/>
+
+<img src = mobileAssignKeysSendInvite.png
+    alt = "assign keys and send invite (Screenshot)">
+
+6. You should be returned to the chat screen. You should now see the wallet config notification says "Pending finalization". On any of your devices you can open the config again by clicking the "View Config" and then the "Finalize wallet" button!<br/><br/>
+
+<img src = mobileAllKeysAssigned.png
+    alt = "mobile phone chat screen (Screenshot)">
+
+7. You will now be able to see the finalize button. When you click that button you will see all of the keys associated with the wallet and other important information. Click the "Finalize wallet" button once you're sure everything is correct.<br/><br/>
+
+<img src = desktopFinalizeWalletConfig.png
+    alt = "desktop wallet config (Screenshot)">
+
+8. You're all done! You should see your wallet on the main screen.<br/><br/>
+
+{{< notice tip >}}
+It is highly recommended to back up your wallet (noted in red, above). There are 2 options for recovery. Either (a) the BSMS file AND enough keys to satisfy the number of signatures requirement or (b) all keys and brute-forcing to "guess" what the original BSMS file was (this can fail with highly-customized and more advanced wallet configurations).If you do not have a backup or a working wallet, you risk losing access to your funds forever.
+{{< /notice>}}
+
+
+{{< /tab >}}
+{{</ tabs >}}
+
+Typically for collaborative wallets, the rule is that the creator of the wallet has to be the one to finalize it. However, for self-managed multisig, the wallet can be finalized from any device (since it's the same user). Once you have added all the required keys to your wallet, you will see a "Finalize wallet" button in the chat screen. You can click that on any device and finalize the wallet! Be sure to look for the backup option in the chat screen and <u>make a backup!</u>
```

Comment:
Typically for collaborative wallets, the rule is that the creator of the wallet must be the one finalizing it. However, for self-managed multisig, the wallet can be finalized from any device (since it's the same user). Once you have assigned all keys to your wallet, you will see a "Finalize wallet" button in the chat screen. You can click that on any device and finalize the wallet! Be sure to follow the instructions to <u>create a backup</u>!